mist-core/src/main/java/edu/snu/mist/core/rpc/DefaultMasterToTaskMessageImpl.java

```diff
@@ -38,17 +44,30 @@ private DefaultMasterToTaskMessageImpl(final GroupAllocationTable groupAllocatio
   }
 
   @Override
-  public Double getTaskLoad() {
-    // The list of event processors
-    // TODO: Synchronize eventProcessor update with master load update.
-    final List<EventProcessor> epList = groupAllocationTable.getKeys();
+  public TaskStats getTaskStats() {
+    final List<Group> groupList = new ArrayList<>();
+    final int numEventProcessors = groupAllocationTable.getKeys().size();
```

Comment:
final List<EventProcessor> eps = groupAllocationTable.getKeys();
final int numEventProcessors = eps.size():

In line 52: 
for (final EventProcessor ep : eps) { 
...

Reply:
I will use `entrySet()` instead.